clippy_lints/src/derivable_impls.rs

```diff
@@ -0,0 +1,114 @@
+use clippy_utils::diagnostics::span_lint;
+use clippy_utils::{in_macro, is_automatically_derived, is_default, match_def_path, paths};
+use rustc_hir::{Block, Expr, ExprField, ExprKind, Impl, ImplItemKind, Item, ItemKind, Node, QPath};
+use rustc_lint::{LateContext, LateLintPass};
+use rustc_session::{declare_lint_pass, declare_tool_lint};
+
+declare_clippy_lint! {
+    /// ### What it does
+    /// Checks for double comparisons that could be simplified to a single expression.
```

Comment:
Looks like all the docs are copied from another lint.

Reply:
Ah, sorry, I forgot to write docs.